client/components/build/LeftSidebar/sections/Section.tsx

```diff
@@ -42,21 +48,42 @@ const Section: React.FC<Props> = ({
 
   const handleAdd = () => {
     const id = path.split('.')[1];
-    const modal: ModalName = validate(id) ? 'builder.sections.custom' : `builder.${path}`;
+    let modal: ModalName = validate(id) ? 'builder.sections.custom' : `builder.${path}`;
 
+    if (type) {
+      modal = `builder.sections.${type}`;
```

Comment:
Find modal by type instead of path. This way we can have one modal "**builder.sections.work**" that works with paths "**builder.sections.work-1, work-2**" and so on